### PR TITLE
pre-workshop survey Japanese

### LINKED ISSUE #1: Translation: pre-workshop/pre-workshop-ja.md
Translation: `pre-workshop/pre-workshop-ja.md`
-----

* Please translate the content [of this file](https://github.com/swcarpentry-ja/assessment-archives/blob/master/pre-workshop/pre-workshop-ja.md) into Japanese.
* You don't need to translate the country list (it will be translated automatically with code)
* Please add a comment to the issue **before** you start translating, so other people know that you are translating.
* It is fine to comment that you will translate part of the file (it is not necessary to translate it all at once).
* Once you have finished translating, send a pull request and reference this issue.

-----
* [このフィアる](https://github.com/swcarpentry-ja/assessment-archives/blob/master/pre-workshop/pre-workshop-ja.md)の内容を日本語に翻訳してください。
* 国のリストは翻訳する必要はありません（コードで自動的に翻訳されます）。
* あなたが翻訳し始めたと分かるように、翻訳を**始める前**このイシューにコメントを書いてください。
* コメントでこのファイルの一部を翻訳しますと書いても大丈夫です（全部一気に翻訳する必要はありません）。
* 翻訳が終わったら、プルリクエストを開いて、このイシューを参照・リンクして下さい。

### PR DESCRIPTION
**Pull request by masamiy**
It is not my masterpiece, but that is the best I could do in a quite limited time. Heavily relying on a reviewer. Just wishing it helps in anyway.

### CODE REVIEW

**Comment by joelnitta** [comment]
Thank you very much! I will try to review it soon.

File: pre-workshop/pre-workshop-ja.md [hunk 1]
```diff
@@ -8,182 +8,178 @@
...
+- データカーペントリー エコロジー 
+- データカーペントリー 経済学 
+- データカーペントリー ジェノミクス 
+- データカーペントリー 地理学 
+- データカーペントリー 社会科学 
+- ライブラリカーペントリー 
+- ソフトウェアカーペントリー 
+- 分からない
+
+### 固有の ID を以下のように入力してください。兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（小文字）＋現在住んでいる通りの最初の3文字 (小文字)
```
Comment (TomKellyGenetics) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
### 固有の ID を以下のように入力してください。兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（小文字）＋現在住んでいる通りの最初の3文字 (ローマ字小文字)。
```

Update for consistency with #4. @naganeayu suggested to specify latin alphabet inputs to ensure alphanumeric IDs without CJK characters (e.g., Kanji, Kana).

File: pre-workshop/pre-workshop-ja.md [hunk 2]
```diff
@@ -8,182 +8,178 @@
...
+- carpentries.org、datacarpentry.org、または software-carpentry.org のウェブサイトでこのワークショップを見た。 
+- アドヴァイザー、もしくはスーパーバイザーにこのワークショップに参加するよう言われた。 
+- 友人もしくは同僚からこのワークショップにについて聞いた。 
+- ソーシャルメディア (Twitter、Facebook、など)でこのワークショップを見た。 
+- 学会、会議、またはセミナーでこのワークショップについて聞いた。 
+- 資金を提供している組織、もしくはプログラムオフィサーからこのワークショップについて聞いた。 
+
+### 以下の質問は任意です。 もし回答したくない場合はあなたのキーボードの下向き矢印押下、もしくは画面をスクロールして次の質問に進んでください。以下のステートメントの合意レベルを教えてください。
+                       
+#### オリジナルの素データへのアクセスは解析を繰り返すために大切です。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: A change to make this identical wording as what I've suggested for the post-workshop survey
```suggestion
#### 元の生データにアクセスできることは、分析を再現するために重要である。   
```

File: pre-workshop/pre-workshop-ja.md [hunk 3]
```diff
@@ -241,7 +237,7 @@ If you have any questions about your rights or treatment as a research participa
 </tbody>
 </table>
 
-#### I know how to search for answers to my technical questions online.  
+#### 技術上の質問にオンラインで回答を検索することが出来る。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: Edit to make this identical to the post-workshop survey:
```suggestion
#### 技術的な質問の答えをオンラインで探す方法が分かる。
```

File: pre-workshop/pre-workshop-ja.md [hunk 4]
```diff
@@ -270,7 +266,7 @@ If you have any questions about your rights or treatment as a research participa
 </tbody>
 </table>
 
-#### While working on a programming project, if I got stuck, I can find ways of overcoming the problem.  
+#### プログラミングプロジェクトにおいて、困難に当たった際、その問題を解決する手法を見つけることが出来る。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: Small suggested change, would also change post-workshop survey to match.
```suggestion
#### プログラミングをしていて行き詰ったときに、その問題を解決する手法を見つけることが出来る。  
```

File: pre-workshop/pre-workshop-ja.md [hunk 5]
```diff
@@ -299,7 +295,7 @@ If you have any questions about your rights or treatment as a research participa
 </tbody>
 </table>
 
-#### I am confident in my ability to make use of programming software to work with data.  
+#### データを使用するためにプログラミングソフトウェアを使用することに自信がある。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: The original mentions "my ability"
```suggestion
#### 自分のプログラミングソフトウェアによるデータ解析能力に自信がある。
```

File: pre-workshop/pre-workshop-ja.md [hunk 6]
```diff
@@ -328,7 +324,7 @@ If you have any questions about your rights or treatment as a research participa
 </tbody>
 </table>
 
-#### Using a programming language (like R or Python) can make my analyses easier to reproduce.   
+#### プログラミング言語 (R やパイソンなど) を使用することは私の解析の再現性を容易にしてくれる。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: Change to make identical to post-workshop survey
```suggestion
#### プログラミング言語（RやPythonなど）を使うことで、自分の分析をより再現しやすくすることができる。
```

File: pre-workshop/pre-workshop-ja.md [hunk 7]
```diff
@@ -8,182 +8,178 @@
...
+   - はい
+   - いいえ
+
+### 18歳以上の方ですか？ *
+
+   - はい
+   - いいえ
+
+### ワークショップのIDは何ですか？YYYY-MM-DD-sitename 形式で、ワークショップのウェブサイトまたは使用した Etherpad の URL に使用されています。以下のテキストボックスにワークショップ ID をコピーして貼り付けてください。
+(短文による回答)
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: Don't need to translate
```suggestion
(short text answer)
```

File: pre-workshop/pre-workshop-ja.md [hunk 8]
```diff
@@ -8,182 +8,178 @@
...
+
+### どのワークショップに参加しますか？ 
+
+- データカーペントリー エコロジー
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: To make it similar with others using kanji
```suggestion
- データカーペントリー 生態学
```

File: pre-workshop/pre-workshop-ja.md [hunk 9]
```diff
@@ -8,182 +8,178 @@
...
+   - いいえ
+
+### ワークショップのIDは何ですか？YYYY-MM-DD-sitename 形式で、ワークショップのウェブサイトまたは使用した Etherpad の URL に使用されています。以下のテキストボックスにワークショップ ID をコピーして貼り付けてください。
+(短文による回答)
+
+### どのワークショップに参加しますか？ 
+
+- データカーペントリー エコロジー 
+- データカーペントリー 経済学 
+- データカーペントリー ジェノミクス
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- データカーペントリー ゲノム解析
```

File: pre-workshop/pre-workshop-ja.md [hunk 10]
```diff
@@ -8,182 +8,178 @@
...
+- データカーペントリー ジェノミクス 
+- データカーペントリー 地理学 
+- データカーペントリー 社会科学 
+- ライブラリカーペントリー 
+- ソフトウェアカーペントリー 
+- 分からない
+
+### 固有の ID を以下のように入力してください。兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（小文字）＋現在住んでいる通りの最初の3文字 (小文字)
+
+(短文による回答)
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: Don't need to translate
```suggestion
(short text answer)
```

File: pre-workshop/pre-workshop-ja.md [hunk 11]
```diff
@@ -8,182 +8,178 @@
...
+
+- 農業もしくは環境科学 
+- 生物医学もしくは健康科学
+- 化学
+- 土木、機械、化学、もしくは原子力工学
+- コンピュータサイエンスもしくは電気工学 
+- 地球科学 
+- 経済学もしくはビジネス 
+- 教育学 
+- 遺伝学、ジェノミクス、もしくはバイオインフォマティクス
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- 遺伝学、ゲノム解析、もしくはバイオインフォマティクス 
```

File: pre-workshop/pre-workshop-ja.md [hunk 12]
```diff
@@ -8,182 +8,178 @@
...
+- 経済学もしくはビジネス 
+- 教育学 
+- 遺伝学、ジェノミクス、もしくはバイオインフォマティクス 
+- ハイパフォーマンスコンピュータ 
+- 人文科学
+- 図書館学もしくは情報科学 
+- 生命科学 
+- 数学もしくは統計学 
+- 医学
+- 有機体生物学 (エコロジー、植物学、動物学、微生物学など)
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- 有機体生物学 (生態学、植物学、動物学、微生物学など) 
```

File: pre-workshop/pre-workshop-ja.md [hunk 13]
```diff
@@ -8,182 +8,178 @@
...
-
-### The next set of questions is optional. If you prefer no to answer a question, please click the down arrow on your keyboard, or scroll down to navigate to the next question.                       Please rate your level of agreement with the following statements.  
-
-#### Having access to the original, raw data is important to be able to repeat an analysis.  
+- ウィンドウズ 
+- 分からない 
+
+### 以下を使用する頻度を教えてください。                                                              
+
+#### ポイントアンドクリックグラフィックユーザーインターフェースに特化したソフトウェア (例: 統計解析: SPSS、SAS、など; 地理学的解析: ArcGIS、 QGIS、など; ジェノミクス解析: Geneious、など)
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
#### ポイントアンドクリックグラフィックユーザーインターフェースに特化したソフトウェア (例: 統計解析: SPSS、SAS、など; 地理学的解析: ArcGIS、 QGIS、など; ゲノム解析: Geneious、など) 
```

File: pre-workshop/pre-workshop-ja.md [hunk 14]
```diff
@@ -8,182 +8,178 @@
...
+#### コマンドシェル (通常マック OS やウィンドウズのパワーシェルを介してターミナルにアクセス) 
+
+- まったくない 
+- 1年に1度以下 
+- 1年に複数回程度 
+- 毎月 
+- 毎週 
+- 毎日 
+
+### あなたの現在のデータマネジメントと解析に関するワークフロー(どのようにデータを収集、オーガナイズ、保存そして解析するか)の満足度を教えてください。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
### あなたの現在のデータマネジメントと解析に関するワークフロー(どのようにデータを収集、整理、保存そして解析するか)の満足度を教えてください。 
```

File: pre-workshop/pre-workshop-ja.md [hunk 15]
```diff
@@ -8,182 +8,178 @@
...
+- 現在の仕事に使用できるスキル習得のため。 
+- 将来の仕事に使用できるスキル習得のため。
+- 仕事の獲得もしくはプロモーションに役立つスキル習得のため。
+- 自分のプログラムや現在の仕事に必要なため。 
+
+### このワークショップをどのようにして知りましたか？ 
+
+- ワークショップに関する E メールもしくはチラシを受け取った。 
+- ニュースレターもしくは大学のウェブサイトでこのワークショップについて読んだ。 
+- carpentries.org、datacarpentry.org、または software-carpentry.org のウェブサイトでこのワークショップを見た。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- carpentries.org、datacarpentry.org、または software-carpentry.org のウェブサイトでこのワークショップの発表を見た。 
```

File: pre-workshop/pre-workshop-ja.md [hunk 16]
```diff
@@ -8,182 +8,178 @@
...
+- 将来の仕事に使用できるスキル習得のため。
+- 仕事の獲得もしくはプロモーションに役立つスキル習得のため。
+- 自分のプログラムや現在の仕事に必要なため。 
+
+### このワークショップをどのようにして知りましたか？ 
+
+- ワークショップに関する E メールもしくはチラシを受け取った。 
+- ニュースレターもしくは大学のウェブサイトでこのワークショップについて読んだ。 
+- carpentries.org、datacarpentry.org、または software-carpentry.org のウェブサイトでこのワークショップを見た。 
+- アドヴァイザー、もしくはスーパーバイザーにこのワークショップに参加するよう言われた。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- 指導教官、もしくはスーパーバイザーからこのワークショップについて聞いた。
```

File: pre-workshop/pre-workshop-ja.md [hunk 17]
```diff
@@ -8,182 +8,178 @@
...
+- 自分のプログラムや現在の仕事に必要なため。 
+
+### このワークショップをどのようにして知りましたか？ 
+
+- ワークショップに関する E メールもしくはチラシを受け取った。 
+- ニュースレターもしくは大学のウェブサイトでこのワークショップについて読んだ。 
+- carpentries.org、datacarpentry.org、または software-carpentry.org のウェブサイトでこのワークショップを見た。 
+- アドヴァイザー、もしくはスーパーバイザーにこのワークショップに参加するよう言われた。 
+- 友人もしくは同僚からこのワークショップにについて聞いた。 
+- ソーシャルメディア (Twitter、Facebook、など)でこのワークショップを見た。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- ソーシャルメディア (Twitter、Facebook、など)でこのワークショップの発表を見た。 
```

File: pre-workshop/pre-workshop-ja.md [hunk 18]
```diff
@@ -8,182 +8,178 @@
...
+- ワークショップに関する E メールもしくはチラシを受け取った。 
+- ニュースレターもしくは大学のウェブサイトでこのワークショップについて読んだ。 
+- carpentries.org、datacarpentry.org、または software-carpentry.org のウェブサイトでこのワークショップを見た。 
+- アドヴァイザー、もしくはスーパーバイザーにこのワークショップに参加するよう言われた。 
+- 友人もしくは同僚からこのワークショップにについて聞いた。 
+- ソーシャルメディア (Twitter、Facebook、など)でこのワークショップを見た。 
+- 学会、会議、またはセミナーでこのワークショップについて聞いた。 
+- 資金を提供している組織、もしくはプログラムオフィサーからこのワークショップについて聞いた。 
+
+### 以下の質問は任意です。 もし回答したくない場合はあなたのキーボードの下向き矢印押下、もしくは画面をスクロールして次の質問に進んでください。以下のステートメントの合意レベルを教えてください。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: Make the same as the post-workshop survey
```suggestion
### 次の質問は任意です。質問に答えたくない場合は、キーボードの下向き矢印をクリックするか、下にスクロールして次の質問に移動してください。 次の記述に対してどの程度同意するかを評価してください。
```

File: pre-workshop/pre-workshop-ja.md [hunk 19]
```diff
@@ -196,11 +192,11 @@ If you have any questions about your rights or treatment as a research participa
  </thead>
 <tbody>
   <tr>
-   <td style="text-align:center;"> Strongly Disagree </td>
+   <td style="text-align:center;"> まったく同意しない </td>
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: I'm not going to make all the individual edits here, but could you change these throughout to be the same as the post-workshop survey?
```suggestion
   <td style="text-align:center;"> 全くそう思わない </td>
```

File: pre-workshop/pre-workshop-ja.md [hunk 20]
```diff
@@ -196,11 +192,11 @@ If you have any questions about your rights or treatment as a research participa
...
    <td style="text-align:center;">  </td>
-   <td style="text-align:center;"> Neutral </td>
+   <td style="text-align:center;"> 何とも言えない </td>
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
   <td style="text-align:center;"> どちらでもない </td>
```

File: pre-workshop/pre-workshop-ja.md [hunk 21]
```diff
@@ -196,11 +192,11 @@ If you have any questions about your rights or treatment as a research participa
...
    <td style="text-align:center;">  </td>
-   <td style="text-align:center;"> Strongly Agree </td>
+   <td style="text-align:center;"> まったく同意する </td>
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
   <td style="text-align:center;"> 強くそう思う </td>
```

File: pre-workshop/pre-workshop-ja.md [hunk 22]
```diff
@@ -357,7 +353,7 @@ If you have any questions about your rights or treatment as a research participa
 </tbody>
 </table>
 
-#### Using a programming language (like R or Python) can make me more efficient at working with data.   
+#### プログラミング言語 (R やパイソンなど) を使用することは私の解析の効率を高めてくれる。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: Make the same as the post-workshop survey
```suggestion
#### プログラミング言語（RやPythonなど）を使用することで、データの処理をより効率的に行うことができる。
```

File: pre-workshop/pre-workshop-ja.md [hunk 23]
```diff
@@ -386,11 +382,11 @@ If you have any questions about your rights or treatment as a research participa
 </tbody>
 </table>
 
-### Please share what you most hope to learn from participating in this workshop. 
+### このワークショップから学びたいことを教えてください。 
 
-(long text answer)
+(長文による回答)
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: No need to translate
```suggestion
(long text answer)
```

File: pre-workshop/pre-workshop-ja.md [hunk 24]
```diff
@@ -642,34 +638,34 @@ If you have any questions about your rights or treatment as a research participa
...
 
-### To which gender do you most identify? 
+### 性別を教えてください。 
 
-- Female 
-- Gender variant/non-conforming 
-- Male 
-- Prefer not to say 
+- 女性 
+- 性別バリアント / 不適合
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: This is important, and we should prioritize a culturally appropriate translation over a literal one. Here is what [yahoo uses](https://about.yahoo.co.jp/info/blog/20180613/diversity.html).
```suggestion
- その他
```

File: pre-workshop/pre-workshop-ja.md [hunk 25]
```diff
@@ -642,34 +638,34 @@ If you have any questions about your rights or treatment as a research participa
...
+### 性別を教えてください。 
 
-- Female 
-- Gender variant/non-conforming 
-- Male 
-- Prefer not to say 
+- 女性 
+- 性別バリアント / 不適合 
+- 男性     
+- 答えたくない
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- 回答しない
```

File: pre-workshop/pre-workshop-ja.md [hunk 26]
```diff
@@ -642,34 +638,34 @@ If you have any questions about your rights or treatment as a research participa
...
-- Asian (Having origins in any of the original peoples of the Far East, Southeast Asia, or the Indian subcontinent including, for example, Cambodia, China, India, Japan, Korea, Malaysia, Pakistan, Indonesia, the Philippine Islands, Thailand, and Vietnam.) 
-- Black or African American (Having origins in any of the Black racial groups of Africa – includes Caribbean Islanders and others of African origin.) 
-- Hispanic or Latino(a) (A person of Spanish-speaking origin or ancestry and/or Latin American origin or ancestry – includes Portuguese and Brazilians.) 
-- Native Hawaiian or Other Pacific Islander (Having origins in any of the original peoples of Hawaii, Guam, Samoa, or other Pacific Islands.) 
-- White (Having origins in any of the original peoples of Europe, the Middle East, or North Africa.) 
-- I prefer not to say. 
+- アメリカンインディアン、もしくはアラスカネイティブ (南北中米のネイティブ起源に持つ、もしくは部族やコミュニティに属する) 
+- アジア人 (極東、東南アジアもしくはカンボジア、中国、インド、日本、韓国、マレーシア、パキスタン、インドネシア、フィリピン、タイおよびベトナムを含むインド亜大陸などを含む)　の原住民のいずれかに起源を持つ。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- アジア系 (極東、東南アジアもしくはカンボジア、中国、インド、日本、韓国、マレーシア、パキスタン、インドネシア、フィリピン、タイおよびベトナムを含むインド亜大陸などを含む)　の原住民のいずれかに起源を持つ。
```

File: pre-workshop/pre-workshop-ja.md [hunk 27]
```diff
@@ -642,34 +638,34 @@ If you have any questions about your rights or treatment as a research participa
...
+### ご自身をどのように説明しますか？以下から 1 つ以上選んでください。  
 
-- American Indian or Alaska Native (Having origins in any of the original peoples of North and South America (including Central America), and who maintains a tribal affiliation or community attachment.) 
-- Asian (Having origins in any of the original peoples of the Far East, Southeast Asia, or the Indian subcontinent including, for example, Cambodia, China, India, Japan, Korea, Malaysia, Pakistan, Indonesia, the Philippine Islands, Thailand, and Vietnam.) 
-- Black or African American (Having origins in any of the Black racial groups of Africa – includes Caribbean Islanders and others of African origin.) 
-- Hispanic or Latino(a) (A person of Spanish-speaking origin or ancestry and/or Latin American origin or ancestry – includes Portuguese and Brazilians.) 
-- Native Hawaiian or Other Pacific Islander (Having origins in any of the original peoples of Hawaii, Guam, Samoa, or other Pacific Islands.) 
-- White (Having origins in any of the original peoples of Europe, the Middle East, or North Africa.) 
-- I prefer not to say. 
+- アメリカンインディアン、もしくはアラスカネイティブ (南北中米のネイティブ起源に持つ、もしくは部族やコミュニティに属する)
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- アメリカンインディアン、もしくはアラスカネイティブ (南北中米の先住民に起源を持つ、もしくは部族やコミュニティに属する) 
```

File: pre-workshop/pre-workshop-ja.md [hunk 28]
```diff
@@ -642,34 +638,34 @@ If you have any questions about your rights or treatment as a research participa
...
-- Asian (Having origins in any of the original peoples of the Far East, Southeast Asia, or the Indian subcontinent including, for example, Cambodia, China, India, Japan, Korea, Malaysia, Pakistan, Indonesia, the Philippine Islands, Thailand, and Vietnam.) 
-- Black or African American (Having origins in any of the Black racial groups of Africa – includes Caribbean Islanders and others of African origin.) 
-- Hispanic or Latino(a) (A person of Spanish-speaking origin or ancestry and/or Latin American origin or ancestry – includes Portuguese and Brazilians.) 
-- Native Hawaiian or Other Pacific Islander (Having origins in any of the original peoples of Hawaii, Guam, Samoa, or other Pacific Islands.) 
-- White (Having origins in any of the original peoples of Europe, the Middle East, or North Africa.) 
-- I prefer not to say. 
+- アメリカンインディアン、もしくはアラスカネイティブ (南北中米のネイティブ起源に持つ、もしくは部族やコミュニティに属する) 
+- アジア人 (極東、東南アジアもしくはカンボジア、中国、インド、日本、韓国、マレーシア、パキスタン、インドネシア、フィリピン、タイおよびベトナムを含むインド亜大陸などを含む)　の原住民のいずれかに起源を持つ。
+- 黒人、もしくはアフリカ系アメリカ人 (アフリカの黒人系グループ　– カリブ諸島や他のアフリカ系の起源を持つ。) 
+- ヒスパニック系もしくはラテン系 (スペイン語を話すもしくは起源を持つ/ ポルトガルおよびブラジルを含むラテン系を起源に持つ）
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- ヒスパニック系もしくはラテン系 (スペイン語を話すもしくは起源を持つ/ ポルトガルおよびブラジルを含むラテン系に起源を持つ）
```

File: pre-workshop/pre-workshop-ja.md [hunk 29]
```diff
@@ -642,34 +638,34 @@ If you have any questions about your rights or treatment as a research participa
...
+- ハワイ原住民もしくは他の太平洋諸島人 (ハワイ、グアム、サモア、もしくは他の太平洋諸島に起源を持つ) 
+- 白人 (ヨーロッパ、中東もしくは北アフリカを起源に持つ。）
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- 白人 (ヨーロッパ、中東もしくは北アフリカに起源を持つ。）
```

File: pre-workshop/pre-workshop-ja.md [hunk 30]
```diff
@@ -642,34 +638,34 @@ If you have any questions about your rights or treatment as a research participa
...
+- 答えたくない
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]:
```suggestion
- 回答しない
```

File: pre-workshop/pre-workshop-ja.md [hunk 31]
```diff
@@ -642,34 +638,34 @@ If you have any questions about your rights or treatment as a research participa
...
-- Gender variant/non-conforming 
-- Male 
-- Prefer not to say 
+- 女性 
+- 性別バリアント / 不適合 
+- 男性     
+- 答えたくない 
 
-### How would you describe yourself? Choose one or more of the following groups.  
+### ご自身をどのように説明しますか？以下から 1 つ以上選んでください。
```
Comment (joelnitta) [inline pre-workshop/pre-workshop-ja.md]: Not very literal, but this is the best I could come up with (I don't think 説明する is appropriate)
```suggestion
### ご自身の民族的背景について、以下から一つ以上選んでください。  
```

**Review by joelnitta — COMMENTED** [review]
Thanks for the translation! Please see my suggestions.
Note: some of the "do you agree" questions should be identical between the pre- and post- surveys. I have made suggested changes on #4 so they should be the same, if my changes are implemented.

File: pre-workshop/pre-workshop-ja.md (hunk 18 above)
Comment (masamiy) [reply pre-workshop/pre-workshop-ja.md]: As it is a keyboard, it does not 'click'.

File: pre-workshop/pre-workshop-ja.md (hunk 15 above)
Comment (masamiy) [reply pre-workshop/pre-workshop-ja.md]: I committed this, but perhaps "このワークショップについて見た。” is better than ”発表". Ditto for other "発表".

File: pre-workshop/pre-workshop-ja.md (hunk 18 above)
Comment (joelnitta) [reply pre-workshop/pre-workshop-ja.md]: Is 「あなたのキーボードの下向き矢印押下」correct though? Seems like it's missing an 「を」
```suggestion
### 以下の質問は任意です。 もし回答したくない場合はあなたのキーボードの下向き矢印を押下して、もしくは画面をスクロールして次の質問に進んでください。以下のステートメントの合意レベルを教えてください。
```

**Comment by joelnitta** [comment]
There are still a few points that didn’t get fixed, but I am going to go ahead and merge the PRs, then fix them myself. I think that will be faster than going back and forth more with the review process.

**Pull request merged**